api/database/sampleApp.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
+from mongoengine import *
+from pymongo.errors import DuplicateKeyError
+from users import Users
+
+app = Flask(__name__)
+
+@app.route('/users', methods = ["GET"])
+def get_users():
+    name = request.args.get('Name')
```

Comment:
Is there any checking that needs to be done on this get call so we don't end up crashing on args.get calls? What happens if it doesn't exist?

Reply:
It won't crash, if the api call doesn't have a key called Name the args.get will return None. Checking if the values are None or not happen after.